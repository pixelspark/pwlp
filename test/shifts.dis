0000.	e0	USER	get_length
0001.	20	PEEKB	0
0002.	11	PUSHB	[01]
0004.	8e	BINARY	SHL
0005.	21	PEEKB	1
0006.	11	PUSHB	[04]
0008.	8e	BINARY	SHL
0009.	22	PEEKB	2
0010.	74	UNARY	SHL8
0011.	23	PEEKB	3
0012.	11	PUSHB	[01]
0014.	8f	BINARY	SHR
0015.	24	PEEKB	4
0016.	11	PUSHB	[04]
0018.	8f	BINARY	SHR
0019.	25	PEEKB	5
0020.	75	UNARY	SHR8
0021.	26	PEEKB	6
0022.	11	PUSHB	[01]
0024.	81	BINARY	SUB
0025.	26	PEEKB	6
0026.	74	UNARY	SHL8
0027.	86	BINARY	OR
0028.	25	PEEKB	5
0029.	74	UNARY	SHL8
0030.	74	UNARY	SHL8
0031.	86	BINARY	OR
0032.	24	PEEKB	4
0033.	74	UNARY	SHL8
0034.	74	UNARY	SHL8
0035.	74	UNARY	SHL8
0036.	86	BINARY	OR
0037.	e3	USER	set_pixel
0038.	01	POP	1
0039.	e4	USER	blit
0040.	07	POP	7

