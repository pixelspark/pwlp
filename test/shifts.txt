a=get_length;
b=a<<1;
c=a<<4;
d=a<<8;
e=a>>1;
f=a>>4;
g=a>>8;
set_pixel(a-1, b, c, d);
blit;