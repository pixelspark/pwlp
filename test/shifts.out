000000 000000 000000 000000 000000 000000 000000 000000 000000 14a000 
